src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -75,6 +78,22 @@ def _create_ui_components(self):
                 schema=element,
             )
 
+    def _create_integration_feature(self):
+        # sentry apps must have at least one feature
+        # defaults to 'integrations-api'
+        try:
+            with transaction.atomic():
+                IntegrationFeature.objects.create(
+                    sentry_app=self.sentry_app,
+                )
+        except IntegrityError as e:
+            self.log(
+                extra={
```

Comment:
You don't have to send `extra`. `self.log` will plop the kwargs you pass into `extra`. This is going to end up with an object like `{extra: {extra: {'sentry_app': '...'}}}`. So:

```python
self.log(sentry_app=self.sentry_app.slug, error_message=e.message)
```